### PR TITLE
Updated Set-DbaErrorLog

### PR DESCRIPTION
**Pull request by potatoqualitee**
made some updates to the command and the tests. simplified output and names. updated help to be more accurate. required sqlinstance. added dbasize

note that the parameter validation did not work so i used the native

image

### CODE REVIEW

**Comment by potatoqualitee** [comment]
Simplified output

image

**Comment by potatoqualitee** [comment]
🤔 is there a way to modify the path or is that just sqlwmi?

File: functions/Set-DbaErrorLog.ps1 [hunk 1]
```diff
@@ -35,130 +35,98 @@ function Set-DbaErrorLog {
...
+        [int]$LogCount,
+        [int]$LogSize,
         [switch]$EnableException
     )
     process {
         foreach ($instance in $SqlInstance) {
             Write-Message -Level Verbose -Message "Connecting to $instance"
             try {
-                $server = Connect-SqlInstance -SqlInstance $instance -SqlCredential $SqlCredential
+                $server = Connect-SqlInstance -SqlInstance $instance -SqlCredential $SqlCredential -MinimumVersion 11
```
Comment (wsmelton) [inline functions/Set-DbaErrorLog.ps1]: You cannot use this.

Comment (wsmelton) [reply functions/Set-DbaErrorLog.ps1]: Setting the log file count is supported for any version of SQL Server (2000+). The only thing restricted to 2012+ is the size.

File: functions/Set-DbaErrorLog.ps1 [hunk 2]
```diff
@@ -35,130 +35,98 @@ function Set-DbaErrorLog {
...
+            
+            $currentNumLogs = $server.NumberOfLogFiles
+            $currentLogSize = $server.ErrorLogSizeKb
+            
+            $collection = [PSCustomObject]@{
+                ComputerName              = $server.NetName
+                InstanceName              = $server.ServiceName
+                SqlInstance               = $server.DomainInstanceName
+                LogCount                  = $currentNumLogs
+                LogSize                   = [dbasize]($currentLogSize * 1024)
```
Comment (wsmelton) [inline functions/Set-DbaErrorLog.ps1]: This change would also need to be done to the Get-DbaErrorLogConfig so the output matches.

**Comment by wsmelton** [comment]
The path for errorlog is changed by using `Set-DbaStartupParameter`, that is the only way to change it without going directly to the registry.

File: functions/Get-DbaErrorLogConfig.ps1 [hunk 3]
```diff
@@ -25,7 +25,7 @@ function Get-DbaErrorLogConfig {
 
             Website: https://dbatools.io
             Copyright: (C) Chrissy LeMaire, clemaire@gmail.com
-            License: GNU GPL v3 https://opensource.org/licenses/GPL-3.0
+            License: MIT https://opensource.org/licenses/MIT
```
Comment (wsmelton) [inline functions/Get-DbaErrorLogConfig.ps1]: My bad, got this from template and didn't catch it was old. I'll go update the templates.

Comment (potatoqualitee) [reply functions/Get-DbaErrorLogConfig.ps1]: ty!

**Comment by potatoqualitee** [comment]
got all passes!

**Pull request merged**